### PR TITLE
update build for Go Function Controller

### PR DESCRIPTION
**Pull request by markfisher**

### CODE REVIEW

**Comment by trisberg** [comment]
Why remove the probes? There is a PR adding the in.

Also, `JAVA_TOOL_OPTIONS` can go away

File: README.adoc [hunk 1]
```diff
@@ -61,21 +60,14 @@ From now on, this README assumes you're running commands from the `riff` reposit
...
-
 ==== Point your local Docker environment to the Docker environment running in minikube:
 
 [source, bash]
 ----
 eval $(minikube docker-env)
 ----
 
-==== Build the Function Sidecar, HTTP Gateway and Topic Controller
+==== Build the Function Sidecar, HTTP Gateway, and Topic Controller
```
Comment (trisberg) [inline README.adoc]: Couldn't we add the build of the Function Controller here as well, looks weird to have that as a separate step later now that it's a Go project

Comment (markfisher) [reply README.adoc]: it was originally due to the tests expecting those resources to be created, but if that's no longer a requirement, I agree they should all be together

**Comment by trisberg** [comment]
Squashed and merged as commit de937b3983eaf618c2464f73b17963c8c1962e87

File: README.adoc (hunk 1 above)
Comment (trisberg) [reply README.adoc]: I'll submit a separate PR with some other changes as well

**Pull request closed without merging**